scripts/bindings.py

```diff
@@ -98,27 +102,27 @@ def generic_function(function):
     """
 
     signature = function
-    rtype, function = function.split(' ', 1)
+    rtype, function = function.split(None, 1)
```

Comment:
On Tue, 07 Jun 2016 01:53:58 -0700
João Ventura <notifications@github.com> wrote:
> > @@ -98,27 +102,27 @@ def generic_function(function):
> >      """
> >  
> >      signature = function
> > -    rtype, function = function.split(' ', 1)
> > +    rtype, function = function.split(None, 1)
> 
> ' ' in this case is more explicit than None..

But less robust since ' ' matches exactly one space whereas None matches
\s+

-- 
Mark Summerfield, Qtrac Ltd.
    diffpdfc windows command line PDF comparison tool
        http://www.qtrac.eu/diffpdfc.html